### DISCUSSION TITLE
working of "stop-at-first-match"  and max_requests

### DISCUSSION THREAD

**Discussion by ItshMoh** [projectdiscovery/nuclei-templates#11270] [Q&A] [answered]
I am working on the nuclei template for the CVE-2022-20699. Here we are sending the requests until we get the status code 500. Here is the link to the [POC](https://github.com/puckiestyle/CVE-2022-20699/blob/main/CVE-2022-20699.py).
Now what should be a proper value for max_request in the metadata. I have added stop-at-first-match =true condition. Is this right. 
Here is the [gitst](https://gist.github.com/ItshMoh/fa03400a5a1c72975534e5e62f00c9ed) tfor the template.

**Comment by ehsandeep** [comment, marked as answer]
@ItshMoh `max_request` is automatically generated/updated during workflow runs. You can ignore populating or updating this field.
